.github/workflows/update-deployment.yml

```diff
@@ -10,7 +10,6 @@ on:
         default: 'prod-test'
```

Comment:
```suggestion
        default: 'prod'
```
this feels a little scary to set the default to `prod`, but... I don't know the behavior if it's set to `prod-test`.

Reply:
comment is out of date now since i was able to expand the scope of this pr to removing the workflow altogether